### ISSUE TITLE
Verify addresses during registration

### ISSUE THREAD

**Issue by rinkp** [GEWIS/gewisdb#169]
To prevent returned mail due to invalid addresses, we can implement something such as [postcode-api.xyz](https://postcode-api.xyz/) during registration. I propose we let people add their postal code and then display the street name as PostNL would use it (frontend). 

The existing addresses don't need to be validated since those have been updated by the secretary when mail was returned

![image](https://user-images.githubusercontent.com/17340320/177403476-904502e2-8a9f-4ac0-877d-70d4b207be3f.png)

**Comment by rinkp** [comment]
Or https://lvbag.github.io/BAG-API/Technische%20specificatie/#/Adres/zoek

**Comment by tomudding** [comment]
Good idea but I foresee some problems with international addresses and potentially needing multiple APIs or having to find one which can do (almost) all addresses (which probably is not free).

Additionally, postcode + house number is considered PII and sending it to a "random" API will require some additional checks beforehand to make sure that everything is done correctly.

**Comment by tomudding** [comment]
Related to the Symfony migration. #564 makes `join.gewis.nl` a real host with its own routes and rate limiting rather than a path allowlist regex in nginx, which is the natural place to add address verification during registration. Tracking epic: #555.
